README.md

```diff
@@ -46,6 +92,20 @@ Ziggurat's documentation can be built with `cargo doc --no-deps --open`.
 
 Ziggurat currently uses rust's standard test runner, a simple `cargo test -- --test-threads=1` should suffice. We use the single threaded executor as spinning up multiple test nodes isn't currently supported.
 
+### Logging
+
+By default, the node's `stdout` and `stderr` output will not be visible. One can enable this (in the code) on a per test basis by inserting a call to `log_to_stdout(true)` before starting the node. This can be used to see the node's logs on screen if appropriately configured.
```

Comment:
Would be great to mention `enable_tracing` for synthetic nodes too?

Reply:
> 
> 
> Clippy breakage may be due to [rust-lang/rust-clippy#7422](https://github.com/rust-lang/rust-clippy/issues/7422).

I think it is. It broke for me locally only after `rustup update`.

Reply:
Thanks, I forgot about that!